Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -0,0 +1,1098 @@
+#include "WarpX.H"
+#include <AMReX_Scan.H>
+#include <AMReX_iMultiFab.H>
+#include <AMReX_MultiFab.H>
+
+/**
+* \brief auxiliary function to count the amount of faces which still need to be extended
+*/
+amrex::Array1D<int, 0, 2>
+WarpX::CountExtFaces() {
+    amrex::Array1D<int, 0, 2> sums{0, 0, 0};
+#ifdef AMREX_USE_EB
+
+    for (int idim = 0; idim < AMREX_SPACEDIM; ++idim) {
+        amrex::ReduceOps<amrex::ReduceOpSum> reduce_ops;
+        amrex::ReduceData<amrex::Real> reduce_data(reduce_ops);
```

Comment:
Should the type be `int`?

Reply:
Yep! Thanks for spotting that. I fixed it.